packages/pn-pa-webapp/src/__mocks__/User.mock.ts

```diff
@@ -1,49 +1,34 @@
-import { PartyEntity, ProductEntity } from '@pagopa/mui-italia';
+import { PartyEntityWithUrl } from "@pagopa-pn/pn-commons";
```

Comment:
Why the use of PartyEntityWithUrl and not PartyEntity?

Reply:
Because there isn't the attribute `entityUrl` and this is what the real api should return